package.json

```diff
@@ -14,24 +14,21 @@
   "scripts": {
     "start": "electron example.js",
     "test": "xo && tsd",
-    "build": "swift build --configuration=release && mv .build/release/screen-capture-permissions .",
-    "prepublish": "npm run build"
+    "native_build": "node-gyp rebuild",
+    "install": "prebuild-install || node install.js"
   },
   "dependencies": {
     "electron-util": "^0.13.0",
     "execa": "^2.0.4",
-    "macos-version": "^5.2.0"
+    "macos-version": "^5.2.0",
+    "prebuild-install": "^6.0.0"
   },
   "devDependencies": {
     "electron": "^7.1.0",
+    "prebuild": "^10.0.1",
     "tsd": "^0.11.0",
     "xo": "^0.24.0"
   },
-  "files": [
```

Comment:
This should not be removed.

Reply:
I had to remove the files section, as all files are necessary if no prebuild is available for the respective node API version and architecture. Thus when using the mac-screen-capture-permissions as a dependency, every file including the Objective-C-sources are required. 

This is then very hard to debug, as building the package will work, but when using it as a dependency, it will not (if not all files are there).

Thus I tought it does not make sense to keep the files whitelist if it has to contain all files anyway (and it would just be a stumbling block if another file is added to the repo and not to the whitelist).